src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -4,9 +4,13 @@
 
 * Add HTTP server span attributes for API Gateway triggers
   ([#626](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/626))
+* Removes `AddAWSLambdaConfigurations` method with default configure parameter.
```

Comment:
This is confusing because from the user's perspective it is not removed but just replaced with another method. Probably we don't even need a changelog entry here, unless it's really ABI-breaking

Reply:
In some strange cases it can be treated as ABI-breaking. https://stackoverflow.com/a/9916197
Most cases it is not.